runtime/cudaq/cudaq.cpp

```diff
@@ -408,6 +409,17 @@ void set_random_seed(std::size_t seed) {
   platform.onRandomSeedSet(seed);
 }
 
+/// @brief Note: a seed value of 0 will cause broadcast operations to use
+/// std::random_device (or something similar) as a seed for the PRNGs, so this
+/// will not be repeatable for those operations.
+void set_photonic_random_seed(std::size_t seed) {
```

Comment:
Q: Can we try to reuse the existing `cudaq::set_random_seed` API?
May need to correspondingly update `getSimulator`.

Reply:
I removed `set_photonic_random_seed`